docs/TESTS.md

```diff
@@ -77,21 +40,22 @@ You should not need to edit this file.  The provided recipe assumes that
 your implementation exists in a source file named after
 the exercise.
 
-For instance, the exercise `bob` expects an implementation in `bob.f90`
-file.
+For instance, the exercise `bob` expects an implementation in `bob.f90` file.
 
 **Create your initial implementation files before running CMake.**
-If you do not have file `bob.f90` when running
-CMake for exercise `bob`, then CMake will generate an error about files
-not being found.  **These files can be empty, but they must exist.**
+If you do not have file `bob.f90` when running CMake for exercise `bob`, then CMake will generate an error about files not being found.
+**These files can be empty, but they must exist.**
 
 Using this recipe, CMake can generate a suitable project for your environment
 by running `cmake -G` with a suitable generator and the location of the
 directory for the exercise.  CMake will generate a build script appropriate
 for your operating system.  To keep those generated files separate from
-your exercise source code, it is common to create a directory called `build`
-to hold these generated build files, as well as the compiled code.  This
-will keep your exercise folder uncluttered and tidy.
+your exercise source code, it is common to create a directory called `Debug` or `build`
+to hold these generated build files, as well as the compiled code.
```

Comment:
Can we call it 'build' consistently?  Debug is true, but really it is here for the build, ultimately, right?

Reply:
meh. While the cpp track does use the term "build", I prefer "Debug" and "Release" and "Profiling" or "RelWithDebInfo" which would be typical build directories you would have in a larger project and CMAKE_BUILD_TYPEs, see https://cmake.org/cmake/help/v3.0/variable/CMAKE_BUILD_TYPE.html

DEBUG|RELEASE|RELWITHDEBINFO|MINSIZEREL

So I would prefer to keep the name "Debug".

Reply:
As long as it makes sense.  I did not see the other folder types though, and some opinions were made (prejudicial, I will admit) when I saw the folder.

Reply:
Can we remove the "or" at least?  And refer to it in a way as in "the debug build" and the folder as "Debug" (if it is capitalized) or lowercase if it is not.

Reply:
For this PR I will keep Debug, but change it to build in another PR.